src/main/java/hw6/pages/enteties/JdiUser.java

```diff
@@ -0,0 +1,46 @@
+package hw6.pages.enteties;
+
+import lombok.AllArgsConstructor;
+import lombok.Setter;
+
+import java.util.Objects;
+
+@Setter
+@AllArgsConstructor
```

Comment:
Why do you implement when you use `lombok`
* `toString`
* `equals` 
* `hashCode`

Reply:
I didn't know that Lombok has a EqualsAndHashCode annotations. Changed